src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -247,9 +247,9 @@ private async Task ProcessInbound()
                 {
                     while (_inbound.TryRead(out var msg))
                     {
-                        if (debug)
+                        if (debug && msg.MessageType != MsgType.RpcLog)
```

Comment:
Remove `RpcLog` from this... I know @mathewc has requested this in the past. It's not used much and can be very noisy.

Reply:
YAY!